conf/default.env

```diff
@@ -0,0 +1,58 @@
+APP_NAME="Invoice Ninja"
+APP_ENV=production
+APP_KEY=base64:3E5HVZ/T+VbZu+oA4GNlwnvwFe3jNtkGbR95K0uwr7A=
```

Comment:
This looks like a secret somehow ? Isn't it necessary to generate it during install ?

Reply:
Yeah, you're right. I'll fix it.